public/scripts/Features.jsx

```diff
@@ -162,7 +181,8 @@ const mapStateToProps = (state, ownProps) => {
 
 const ffMapDispatchToProps = dispatch => (
   {
-    groupToggleCheckedFeatures: prefix => dispatch(Action.groupToggleCheckedFeatures(prefix))
+    groupToggleCheckedFeatures: ctgy => dispatch(Action.groupToggleCheckedFeatures(ctgy)),
```

Comment:
hey, that works! I'll remove all the `mapDispatchToProps` stuff from this module for now, and create an issue to do away with it elsewhere, too.